features/step_definitions/auction_create_steps.rb

```diff
@@ -1,5 +1,7 @@
 Given(/^there is an? (.+) auction$/) do |label|
   @auction = case label
+             when 'unpublished'
```

Comment:
Nice! I am considering actually splitting this method into a bunch of separate steps (since it's just a big case method anyway, but that doesn't hold up this PR)